xml/yast2_ncurses.xml

```diff
@@ -205,7 +205,7 @@
   </figure>
  </sect1>
 
- <sect1 xml:id="sec.yast.ncurses_advanced_hotkeys">
+ <sect1 xml:id="sec.yast.text.advanced_hotkeys">
```

Comment:
"hotkeys"->"key_combo" or just "key"? would be more in line with our style. (noun should be in singular too)